### ISSUE TITLE
Can't press on keyboard

### ISSUE THREAD

**Issue by Arturcik22** [Project-Elixir/issue_tracker#48]
### Issue type

User interface

### Exact version and Build date

v4.1 / 2024.02.05

### Bug Description

When I am trying to comment or to type something the keyboard pops up but I can't touch it. I noticed that is I tap below it the first row of letters type but I need to align correctly my finger with them. I really can't stand this bug and it's all about the keyboard.
*I'm using a Redmi Note 9S

### Steps to Reproduce

1.Try to type something
2.The keyboard isn't working

### Acknowledgements

- [X] I have checked that the device is officially supported (for device specific reports and not source related).
- [X] I am running the latest version available on projectelixiros.com for this device.
- [X] I have searched the existing issues and this is a new issue and not a duplicate of any another open issue.
- [X] I have written a short but informative title.
- [X] I have filled out all of the requested information in this issue properly.

**Comment by RahulPalXDA** [comment]
Did you checked that, your touch screen is properly working?

**Comment by Arturcik22** [comment]
> Did you checked that, your touch screen is properly working?
Of course, It was working fine on MIUI, I also noticed right now the same thing happens on TikTok popups like share and other popups

**Comment by ugly-kid-af** [comment]
Fixed in latest Project Elixir v4.8 Update !
